local/check-dependencies.sh

```diff
@@ -34,9 +39,16 @@ function checkDependency() {
   Missing=$(missingCommands $@)
 
   echo -ne "${Name}: "
-  if [ $Missing ]; then failed ; else passed ; fi
+  if [ $Missing ] && [ $AutoInstall ] && [ $AutoInstall == "install" ];
+    then install $Name ;
+  elif [ $Missing ];
+    then failed ;
```

Comment:
Our default syntax in other scripts is for the `then` to be on the preceding line after the semi-colon.